composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "alymosul/laravel-exponent-push-notifications",
+    "name": "codificio/laravel-exponent-push-notifications",
```

Comment:
Should not change that line!

Reply:
Oh I'm sorry. I've committed the fix. Do you prefer a new PR without that commit?